srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -50,6 +50,9 @@
     private static final String HEIGHT_UNITS = "inches";
     private static final String ADL_ENTERPRISE_TITLE = "NURSING ADMISSION EVALUATION NOTE";
     
+    /**
+     * The expected date format for information received from VistA.
+     */
     public static final String VISTA_DATE_OUTPUT_FORMAT = "MM/dd/yy@HH:mm";
```

Comment:
We should probably specify that this is in `SimpleDateFormat`... format. Why not just make it a `SimpleDateFormat` instance in the first place?